docs/docs/building/rollup-plugin-html.md

```diff
@@ -234,6 +269,10 @@ export interface RollupPluginHTMLOptions {
   transformHtml?: TransformHtmlFunction | TransformHtmlFunction[];
   /** Whether to extract and bundle assets referenced in HTML. Defaults to true. */
   extractAssets?: boolean;
+  /** Define a full absolute url to your site (e.g. https://domain.com) */
+  siteUrl?: string;
+  /** Whether to set full absolute urls for ['meta[property=og:image]', 'link[rel=canonical]', 'meta[property=og:url]'] or not. Requires a siteUrl to be set. Default to true. */
+  fullAbsoluteUrls?: boolean;
```

Comment:
```suggestion
  absoluteSocialMediaPaths?: boolean;
```

Reply:
they are urls and not pathes?
how about `absoluteSocialMediaUrls`?

Reply:
used `absoluteSocialMediaUrls`